### ISSUE TITLE
Was unable to start. Microsoft.WindowsAzure.Storage: Service Unavailable`

### ISSUE THREAD

**Issue by lillheaton** [Azure/azure-functions-core-tools#1796]
Running the template for TimerTrigger function fails with 

`The listener for function 'Function1' was unable to start.
The listener for function 'Function1' was unable to start. Microsoft.WindowsAzure.Storage: Service Unavailable`

Azure Functions Core Tools (2.7.2100 Commit hash: 5afacc827c2848e4debc23bb96604f1ffce09cc7)
Function Runtime Version: 2.0.12961.0

local.settings.json is set to default
``` json
{
    "IsEncrypted": false,
    "Values": {
        "AzureWebJobsStorage": "UseDevelopmentStorage=true",
        "FUNCTIONS_WORKER_RUNTIME": "dotnet"
    }
}
```

Running:
Azure Storage Emulator 5.10.0.0
Visual Studio 16.4.4
.NET Core SDK 3.1.101

**Comment by anthonychu** [comment]
Can you check if you can connect to the local emulator with [Azure Storage Explorer](https://azure.microsoft.com/en-us/features/storage-explorer/)?

**Comment by lillheaton** [comment]
@anthonychu, yes I'm able to connect to the emulator with the storage explorer. I can see the queues and storage tables, however the blob I have issues with connecting (503 service unavailable). 

I'm also able to run azure functions with storage queue trigger for example. But not TimeTriggers

**Comment by anthonychu** [comment]
Timer triggers use blobs to ensure only one function app instance runs a given timer. Looks like there’s a problem with blobs in your storage emulator. Can you try resetting or reinstalling the emulator?

**Comment by cparker4486** [comment]
I'm facing a similar issue and I'm hoping the resolution for my issue is the same for this issue's resolution.

This is the error I am getting after running `func host start`.

![image](https://user-images.githubusercontent.com/1189638/74357459-cda66280-4d74-11ea-85f6-b78d2a67e7ce.png)

I did my best to follow your advice of "resetting or reinstalling the emulator". I tried resetting because I had just tried reinstalling before finding this issue.

![image](https://user-images.githubusercontent.com/1189638/74357697-1cec9300-4d75-11ea-80ab-0966dd35a849.png)

I follow security best-practices and do not run my user as a local administrator. Instead, I have a separate account I use when I install software. I don't know if this is the issue or not.

(Core Tools used to be so easy to work with... what happened? I've been struggling with this thing for a few hours now.)
